qabelbox/src/main/java/de/qabel/qabelbox/box/interactor/FileBrowserUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package de.qabel.qabelbox.box.interactor
+
+import de.qabel.qabelbox.box.dto.BoxPath
+import de.qabel.qabelbox.box.dto.BrowserEntry
+import de.qabel.qabelbox.box.dto.DownloadSource
+import de.qabel.qabelbox.box.dto.UploadSource
+import de.qabel.qabelbox.box.provider.DocumentId
+import rx.Observable
+
+interface FileBrowserUseCase {
```

Comment:
the suffix UseCase doesn't fit well to this interface and is thus not very descriptive (imho)